### DISCUSSION TITLE
Image editor - Upgrade

### DISCUSSION THREAD

**Discussion by wmdhosting** [craftcms/cms#12465] [Ideas] [answered]
This can be great and simple upgrade for current image editor.. 
https://pqina.nl/pintura/?ref=editphoto

**Integrates with your favourite stack**
Pintura Image Editor is written in vanilla JavaScript and has zero dependencies. It ships with multiple adapter components to facilitate fast integration with popular frameworks like jQuery, React, Vue, Angular, and Svelte.
https://pqina.nl/pintura/tour/

There is even Video editor extension..

**Comment by brandonkelly** [comment, marked as answer]
That is a third party service, so not something we’d make Craft rely on out of the box. But I could see us eventually making it possible for a plugin to replace the built-in editor with that.
